pubspec.yaml

```diff
@@ -1,25 +1,24 @@
 name: flame
 description: A minimalist Flutter game engine, provides a nice set of somewhat independent modules you can choose from.
-version: 1.0.0-rc7
+version: 1.0.0-rc7-nullsafety.0
 homepage: https://github.com/flame-engine/flame
 
 dependencies:
   flutter:
     sdk: flutter
-  ordered_set: ^2.0.0
-  meta: ^1.1.8
-  vector_math: '>=2.0.0 <3.0.0'
+  ordered_set: ^3.0.0-nullsafety
+  vector_math: '>=2.1.0-nullsafety <3.0.0'
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  test: ^1.9.4
+  test: ^1.16.0-nullsafety
   dart_code_metrics: ^2.4.0
```

Comment:
This one isn't null-safe right?

Reply:
they have not done it yet. they are blocked by dart, apparently.